gke-policies/policy/autopilot_cluster.rego

```diff
@@ -0,0 +1,34 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# METADATA
+# title: GKE Autopilot mode
+# description: GKE Autopilot mode is the recommended way to operate a GKE cluster
+# custom:
+#   group: Management
+#   severity: High
```

Comment:
should it be High severity?

Reply:
agree that it seems to be too high, but it's now the recommended way to run GKE, wdyt? Maybe Medium it's a little bit less "Urgent"?